lib/consyncful/base.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'consyncful/configuration'
```

Comment:
**question:** Will it work without this `require`?

Reply:
not sure, I can give it a go 👍